test/image/mocks/sort_by_value_matching_axes.json

```diff
@@ -0,0 +1,43 @@
+{
+  "data": [{
+    "type": "scatter",
+    "orientation": "v",
+    "x": ["a", "b", "c"],
+    "y": [7, 2, 3],
+
+    "z": [
```

Comment:
We can :hocho: `z` and `dimensions` out of this mock

Reply:
Done in 2d379f7647bec909b806f56276b6f6225578bc6b